store/transfer_processor.go

```diff
@@ -63,21 +64,16 @@ func newTransferProcessor(blockNumber int64,
 	}
 }
 
-func (s *transferProcessor) process(ctx context.Context, events []*model.Transfer, coinbases []gethCommon.Address) (err error) {
+func (s *transferProcessor) process(ctx context.Context, events []*model.Transfer) (err error) {
 	// Update total balance for new subscriptions, map[group][token]balance
 	totalBalances := make(map[int64]map[gethCommon.Address]*big.Int)
 	totalFees := make(map[int64]*big.Int)
+	totalMinerReward := new(big.Int)
+	totalUncleRewards := new(big.Int)
 
 	// Collect modified addresses
 	seenAddrs := make(map[gethCommon.Address]struct{})
 	var addrs [][]byte
-	// From block / uncles coinbases
-	for _, cb := range coinbases {
-		if _, ok := seenAddrs[cb]; !ok {
-			seenAddrs[cb] = struct{}{}
-			addrs = append(addrs, cb.Bytes())
```

Comment:
We need to keep it. We used `addrs` to query subscriptions and the coinbases may be our subscriptions.

Reply:
this condition are handled in https://github.com/getamis/eth-indexer/pull/54/files#diff-778fd13deb3062e86beaee58b3afff70R96